_data/projects/in-toto.yml

```diff
@@ -0,0 +1,11 @@
+name: in-toto
+desc: A software supply chain security framework
+site: https://in-toto.io/
+tags:
+- Python
+- Security
+- Software Supply Chain
+- CI/CD
+upforgrabs:
+  name: up for grabs
```

Comment:
@SantiagoTorres It's "Up for grabs" according to your label. Please correct it.

Reply:
Oh woops! Thanks for the review!